WordPress/Classes/ViewRelated/Domains/Register/RegisterDomainDetails/Views/RegisterDomainDetailsFooterView.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+import WordPressAuthenticator
+
+class RegisterDomainDetailsFooterView: UIView, NibLoadable {
+
+    @IBOutlet weak var nuxButton: NUXButton!
```

Comment:
Can we make this var name more descriptive? This will be confusing as we have NUX buttons everywhere! 😄

Reply:
Sure, I renamed it as submitButton